include/ddprof_defs.hpp

```diff
@@ -33,7 +33,7 @@ inline constexpr int k_size_api_key = 32;
 
 // Maximum number of profiled pids
 // Exceeding a number of PIDs overloads file descriptors and memory
-inline constexpr unsigned kMaxProfiledPids{100};
+inline constexpr unsigned k_max_profiled_pids{100};
```

Comment:
Should we remove this constant since value is now configurable ?

Reply:
It is still convenient to have default parameters for some of the APIs. It avoids thinking about this during unit testing.

Reply:
Then it could be renamed to `k_default_max_profiled_pids`